### PR TITLE
Dev-louis

### PR DESCRIPTION
**Pull request by lindlou**
This pull request is for resolution and framerate, license and layout changes.
I also want to set image and video formats in settings and document shortcuts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,11 @@ Also: you might be able to do real damage with the CLI to all your data,
 because you have the full power of python at your fingertips.
 Enjoy!
 
+### Shortcuts
+Shortcuts for each action are indicated in square brackets in the app(for example `[F]` to set the file name).
+Shortcuts are inactivated when you input a text or a number. Press enter to end input and activate shortcuts again.
+Press `Ctrl+Q` to close the app.
+
```
Comment (trappitsch) [inline README.md:229]: Would be good to tell the user how they can find out on what shortcuts are set. They are implemented in the tooltips, no?

**Review by trappitsch — CHANGES_REQUESTED** [review]
See comment for readme, looks good to me otherwise.

File: README.md (hunk 1 above)
Comment (lindlou) [reply README.md:229]: Not in the tooltips, in the labels. For example 'File name [F]' or 'Capture image [Space]'

Comment (trappitsch) [reply README.md:229]: ah ok, that resolves this questions

**Review by trappitsch — APPROVED** [review]

**Pull request merged**